src/battle-animations.ts

```diff
@@ -1085,12 +1085,22 @@ export class BattleScene implements BattleSceneStub {
 		if (!this.animating) return;
 		const side = this.battle.sides[siden];
 		const spriteIndex = +side.isFar;
+		let x = side.x;
+		let y = side.y;
+		if (this.battle.gameType === 'freeforall') {
+			x += side.isFar ? 20 : -20;
+			if (side.n > 1) {
+				x += (side.isFar ? -140 : 140);
+				y += side.isFar ? 14 : -20;
+			}
+		}
+
 		switch (id) {
 		case 'auroraveil':
 			const auroraveil = new Sprite(BattleEffects.auroraveil, {
 				display: 'block',
-				x: side.x,
-				y: side.y,
+				x,
+				y,
```

Comment:
Don't these still need to be `x: x` `y: y`?

Reply:
They shouldn't need to be since `x` and `y` are variables?

Reply:
Shouldn't they need to be what Marty said since old IE / etc doesn't support that?

Reply:
What's the resulting code when the build step converts it from TypeScript to JavaScript?